ClientApp/src/Api/apiClient.ts

```diff
@@ -0,0 +1,28 @@
+﻿export interface NewSighting {
+  species: string;
+  quantity: string;
+  location: string;
+  latitude: number;
+  longitude: number;
+  description: string;
+  sightedAt: string;
+  submittedByName: string;
+  submittedByEmail: string;
+}
+
+export async function submitSighting(newSighting: NewSighting) {
+  const response = await fetch(
+    `https://localhost:5001/submit-sighting/submit`,
```

Comment:
I think we can remove the https://localhost:5001 and just have the latter path, (make sure to test it still works!) As otherwise this won't work when we deploy it as it's not hosted at localhost:5001 :)

Reply:
resolved